controllers/cluster_controller.go

```diff
@@ -35,7 +37,7 @@ import (
 )
 
 const (
-	defaultRequeueTime = time.Minute
+	defaultRequeueTime = 10 * time.Second
```

Comment:
can we not change this? just have a different const for your new usecase

Reply:
do we want to wait that long before another update on the status? that's what would happen whenever this is used since the status is updated after every reconcile.